lib/order.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'menu'
+require 'rubygems'
+require 'twilio-ruby'
+require 'time'
+# require 'sinatra'
+
+# post '/message' do
+#   number = params['From']
+#   body = params['Body']
+
+#   content_type 'text/xml'
+#   "<Response>
+#     <Message>
+#       Hello #{number}. You said: #{body}
+#     </Message>
+#   </Response>"
+# end
+
+class Order
+
+  attr_reader :order, :order_total, :simp_order
+
+  def initialize
+
+    @menu = Menu.new
+    @order = []
+    @order_total = 0
+    @simp_order = ""
+
```

Comment:
I'd put all the text stuff into its own class. I did have difficulties getting it to work from the order object, but my feeling was since it's a new noun, it should probably be its own class.